src/main/java/frc/robot/commands/ShooterSetRawCMD.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands;
+
+import java.util.function.Supplier;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Shooter2020;
+
+public class ShooterSetRawCMD extends CommandBase {
```

Comment:
I think commands should always get a clear class comment telling you what the command does. I don't know what raw means in this context.